src/xdocs/config_coding.xml

```diff
@@ -4993,6 +4999,24 @@ if (&quot;something&quot;.equals(x))
         <source>
 &lt;module name=&quot;StringLiteralEquality&quot;/&gt;
         </source>
+        <p>
+          Examples of violations:
+        </p>
+        <source>
+String status = "pending";
+
+if (status == "done") {} // violation
+
+while (status != "done") {} // violation
+
+boolean flag = (status == "done"); // violation
+
+boolean flag = (status.equals("done")); // OK
+
+String name = "X";
+if (name == getName()) {} // OK, limitation that check cannot tell runtime type returned
+from method call
```

Comment:
this isn't valid java code. Its a comment so to continue on the next line you have to start with `//`.
Can you move the whole comment to the next line so it takes up only 1 line?